routes/users.js

```diff
@@ -8,5 +10,9 @@ userRoute.put('/change-password/:id', (changeUserPassword));
 userRoute.patch('/:id/:token', (resetUserPassword));
 userRoute.post('/forgot-password', (userForgotPassword));
 userRoute.post('/login', (userLogin));
+//userRoute.get('/', authenticate, (getAllUsers));
+//userRoute.delete('/:userId', authenticate, (terminateUser));
+userRoute.get('/:recipeId', userAuthenticate, (getARecipe));
+userRoute.get('/', userAuthenticate, (getAllRecipess));
```

Comment:
I thought you have recipe.js file in your route folder.
But you are still putting `recipe` endpoint in `user.js` file